awstesting/integration/constants.go

```diff
@@ -0,0 +1,12 @@
+package integration
+
+const (
+	// Byte is 8 bits
+	Byte int64 = 1
+	// KibiByte (KiB) is 1024 Bytes
+	KibiByte = Byte * 1024
+	// MebiByte (MiB) is 1024 KiB
+	MebiByte = KibiByte * 1024
+	// Gibibyte (GiB) is 1024 MiB
+	Gibibyte = MebiByte * 1024
```

Comment:
What do you think about putting these in a more generic location in the SDK so its more easily used. There are several test cases, and even some SDK logic that uses inline usage of n bytes.